CHANGES/8481.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+Fixed the incorrect rejection of ``ws://`` and ``wss://`` urls
+-- by :user:` AraHaan`.
```

Comment:
@AraHaan @bdraco apparently, there was a typo here — a rogue whitespace after the first backtick.